frontend/src/metabase/browse/components/BrowseContainer.styled.tsx

```diff
@@ -28,7 +27,7 @@ export const BrowseHeader = styled.header`
   display: flex;
   flex-direction: column;
   padding: 1rem 2.5rem 3rem 2.5rem;
-  color: ${color("dark")};
+  color: ${({ theme }) => theme.fn.themeColor("dark")};
```

Comment:
If it is in Mantine theme colors, then should we also add a CSS variable for dark?